src/pages/ViewDog/ViewDog.jsx

```diff
@@ -0,0 +1,49 @@
+import { React, useState, useEffect } from 'react';
+import { useNavigate, useParams } from 'react-router-dom';
+import { Button, Input, Avatar, Textarea, Box, Flex } from '@chakra-ui/react';
+import { AddIcon } from '@chakra-ui/icons';
+import BreadcrumbBar from '../../components/BreadcrumbBar/BreadcrumbBar';
+
+const ViewDog = () => {
+  const { id: dogId } = useParams();
+  const [editable, setEditable] = useState(false);
+  const { backend } = useBackend();
+  const Navigate = useNavigate();
+
+  const [dog, setDog] = useState({});
+
+  const handleEditButton = () => {
```

Comment:
🚫 **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'handleEditButton' is assigned a value but never used.